src/backend/application_server.py

```diff
@@ -0,0 +1,133 @@
+
+"""
+Module defining our flask application server.
+"""
+
+import os
+import logging
+from PIL import Image
+from pathlib import Path
+import torch
+import torchvision.transforms as transforms
+from flask import Flask, flash, request, redirect, render_template,\
+                  send_from_directory
+from werkzeug.utils import secure_filename
+from gan import Generator
+from torchvision.utils import save_image
+
+UPLOAD_FOLDER = Path.cwd() / 'uploads/'
```

Comment:
What do you think of instantiating an object containing all of the global state on server start and making said object globally available?
This could then be referenced in upload_file.